test/integration/crcsuite/crcsuite.go

```diff
@@ -274,7 +274,12 @@ func FileExistsInCRCHome(fileName string) error {
 func ConfigFileInCRCHomeContainsKeyMatchingValue(format string, configFile string, condition string, keyPath string, expectedValue string) error {
 
 	if expectedValue == "current bundle" {
-		expectedValue = bundleName
+
+		if bundleEmbedded {
+			expectedValue = filepath.Join(CRCHome, "cache", bundleName)
+		} else {
+			expectedValue = bundleName
+		}
```

Comment:
Most of this is reverted in the next commit?

Reply:
Yes, I tested it on windows in the meantime. I use regex to match the content of a config file against the expected value. If the expected value is a path, the backslashes escape characters inside regex :(. I'll clean up the commits though, thanks.